### PR TITLE
tweak to the perf urls

### PR DESCRIPTION
**Pull request by devoncarew**
- minor change to how we populate perf urls; this will allow the IntelliJ 'unused property file entry' lint help us notice when we are / aren't using url entries

@jacob314

### CODE REVIEW

File: src/io/flutter/FlutterBundle.properties [hunk 1]
```diff
@@ -206,7 +206,6 @@ info.level.title=Info
 warning.level.title=Warning
 severe.level.title=Severe
 shout.level.title=Shout
-# End Flutter Log Color Page
```
Comment (jacob314) [inline src/io/flutter/FlutterBundle.properties]: was deleting this line intentional?

File: src/io/flutter/perf/PerfTipRule.java [hunk 2]
```diff
@@ -45,6 +44,7 @@
     String hackFileName,
     String message,
     String id,
+    String url,
```
Comment (jacob314) [inline src/io/flutter/perf/PerfTipRule.java]: update the comment for id now that you have changed what the id is used for.

File: src/io/flutter/FlutterBundle.properties (hunk 1 above)
Comment (devoncarew) [reply src/io/flutter/FlutterBundle.properties]: Yes - there's a comment a few lines above mentioning the start of this section; I don't think we generally need to bookmark sections in this file (we don't other places).

File: src/io/flutter/perf/PerfTipRule.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/perf/PerfTipRule.java]: done

**Pull request merged**